src/routes/(signed_in)/courses/[courseID]/sections/[sectionID]/+page.server.ts

```
@@ -23,7 +31,133 @@ export const load: PageServerLoad = async ({ locals, cookies, params }) => {
 };
 
 export const actions: Actions = {
-	default: async () => {
-		// TODO: Implement Change Section Member Role
+	updateSection: async ({ locals, request, cookies, params }) => {
+		const { courseID, sectionID } = params;
+		const userID = verifyAuthentication(locals, cookies);
+		await verifyUserIsApartOfInstructionalTeam(cookies, userID, sectionID);
+
+		const data: FormData = await request.formData();
+		const formMaxDailyBookableHours = data.get("max_daily_bookable_hours");
+		if (!formMaxDailyBookableHours) {
+			return fail(400, { maxDailyBookableHours: formMaxDailyBookableHours, missing: true });
+		}
+		if (typeof formMaxDailyBookableHours !== "string") {
+			return fail(400, { maxDailyBookableHours: formMaxDailyBookableHours, invalid: true });
+		}
+		const formSectionNumber = data.get("section_number");
+		if (!formSectionNumber) {
+			return fail(400, { sectionNumber: formSectionNumber, missing: true });
+		}
+		if (typeof formSectionNumber !== "string") {
+			return fail(400, { sectionNumber: formSectionNumber, invalid: true });
+		}
+
+		const maxDailyBookableHours = parseFloat(formMaxDailyBookableHours);
+		if (isNaN(maxDailyBookableHours) || maxDailyBookableHours < 0) {
+			return fail(400, { maxDailyBookableHours, invalid: true });
+		}
+		const sectionNumber = parseInt(formSectionNumber);
+		if (isNaN(sectionNumber) || sectionNumber < 0) {
+			return fail(400, { sectionNumber, invalid: true });
+		}
+
+		const section = await updateSection(sectionID, sectionNumber, maxDailyBookableHours);
+		if (!section) {
+			error(500, `Internal server error: Failed to update section with ID: ${sectionID}`);
```

Nit: Let's throw the error here so we can get better visibility (via stack traces).